### PR TITLE
Added instructions for SUSE-based distributions

### PR DESCRIPTION
**Pull request by rmahique**
Added instructions for SUSE-based distributions

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Hi @rmahique 

I'd expect also to see a change to https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -192,6 +192,36 @@ To upgrade kubectl to another minor release, you'll need to bump the version in
...
+# This overwrites any existing configuration in /etc/zypp/repos.d/kubernetes.repo
+cat <<EOF | sudo tee /etc/zypp/repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
+enabled=1
+gpgcheck=1
+gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+EOF
+```
```
Comment (windsonsea) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:213]:
```suggestion
   ```bash
   # This overwrites any existing configuration in /etc/zypp/repos.d/kubernetes.repo
   cat <<EOF | sudo tee /etc/zypp/repos.d/kubernetes.repo
   [kubernetes]
   name=Kubernetes
   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
   enabled=1
   gpgcheck=1
   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
   EOF
   ```
```

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -192,6 +192,36 @@ To upgrade kubectl to another minor release, you'll need to bump the version in
...
+
+{{< note >}}
+To upgrade kubectl to another minor release, you'll need to bump the version in `/etc/zypp/repos.d/kubernetes.repo` before running `zypper update`. This procedure is described in more detail in [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
+{{< /note >}}
```
Comment (windsonsea) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:217]:
```suggestion
   {{< note >}}
   To upgrade kubectl to another minor release, you'll need to bump the version in `/etc/zypp/repos.d/kubernetes.repo`
   before running `zypper update`. This procedure is described in more detail in
   [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
   {{< /note >}}
```

**Review by windsonsea — COMMENTED** [review]
See [preview](https://deploy-preview-42913--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/). The list is broken since an indentation of 3 spaces is required.

**Comment by windsonsea** [comment]
Hi, please squash your commits into a single one if possible. Refer to: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 3]
```diff
@@ -192,6 +192,36 @@ To upgrade kubectl to another minor release, you'll need to bump the version in
...
+   To upgrade kubectl to another minor release, you'll need to bump the version in `/etc/zypp/repos.d/kubernetes.repo`
+   before running `zypper update`. This procedure is described in more detail in
+   [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
+   {{< /note >}}
+
+1. Install kubectl using `zypper`:
+
+   ```bash
+   sudo zypper install -y kubectl
+
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:224]: Please close the (bash) code block here.

**Comment by sftim** [comment]
@kubernetes/sig-cluster-lifecycle-leads any objection here?

**Review by neolit123 — COMMENTED** [review]
FWIW, we have preferred to not have SUSE instructions in the kubeadm docs before. mostly because SIG release may not add tests for these distros and we didn't have anyone to verify the installations step by atep.

**Comment by sftim** [comment]
@kubernetes/sig-release-leads we'd like your view here too, then.

**Review by saschagrunert — APPROVED** [review]
LGTM other than the review comment from @sftim

**Comment by sftim** [comment]
OK, let's ship this - I'll file an issue about the fixup.

/approve

**Pull request merged**